timemachine/fe/free_energy.py

```diff
@@ -1181,10 +1221,20 @@ def sample_replica(xvb: CoordsVelBox, state_idx: StateIdx) -> Trajectory:
 
             md_params_replica = replace(md_params, n_frames=1, n_eq_steps=0, seed=state_idx + current_frame)
 
-            return sample_with_context(context, md_params_replica, temperature, ligand_idxs, max_buffer_frames=100)
+            assert md_params_replica.n_frames == 1
```

Comment:
Q: is it actually necessary to have `n_frames == 1` anymore, since we're passing `max_buffer_frames=1`?

Reply:
If `md_params.n_frames == 2` and `max_buffer_frames=1` then calling `next()` would be incorrect since it wouldn't be the last frame.

`max_buffer_frames` here doesn't really matter since `n_frames=1`